### DISCUSSION TITLE
Systems other than 5e?

### DISCUSSION THREAD

**Discussion by RealityBlights** [DFreds/dfreds-convenient-effects#174] [General]
Any chance this amazing module could work with, or be made to, systems other than 5e? (Such as Starfinder, which I run)
I've found the CUB module not to my liking after the change from 5e, and when I found this module, I damn near lost my mind. This is a spectacular piece of code. Only problem is, it seems to only function with the big boy on the playground.

**Comment by DFreds** [comment]
It's not as straightforward as it seems, since a lot of functionality relies on assuming 5e is the system. But I absolutely want to do it, I just need to spend the effort

**Reply by Pete1974** [reply]
Would this module work with other systems if there were no effects included ready to use? I would really love to use this module to create my own effect, no need for pre-built ones. Is there a solution?
